packages/protocol/contracts/tokenvault/ERC721Vault.sol

```diff
@@ -234,7 +234,7 @@ contract ERC721Vault is BaseNFTVault, IERC721ReceiverUpgradeable {
                 });
 
                 for (uint256 i; i < op.tokenIds.length; ++i) {
-                    t.transferFrom(user, address(this), op.tokenIds[i]);
```

Comment:
I believe there is no need to use `.safeTransferFrom` here as ERC721 is transferred to this contract which has already implemented `onERC721received`